### PR TITLE
Requirements

### PR DESCRIPTION
**Pull request by jburel**
* remove usage of ``OMERO.server/share/web/requirements**``
* fix incorrect django-redis statement
* remove out-of-date and unused osx steps install
* remove unused requirements file (coming from omero-install)

### CODE REVIEW

**Comment by pwalczysko** [comment]
sorry, just cannot find the right built part of the doc

I thought it is this one -
https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/sysadmins/unix/install-web/web-deployment.html

but the bits of the file text I am trying to search for in the doc above do not match...

**Comment by jburel** [comment]
this is the page
https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/sysadmins/unix/install-web.html

File: omero/sysadmins/unix/install-web.rst [hunk 1]
```diff
@@ -85,7 +85,7 @@ Gunicorn is restarted using ``omero web restart``.
 
   -  OMERO.web offers alternative session backends to automatically delete stale data using the cache session store backend, see :djangodoc:`Django cached session documentation <topics/http/sessions/#using-cached-sessions>` for more details.
 
-  - `Redis 3.3+ <https://redis.io/>`_ requires `django-redis >= 4.10 <https://niwinz.github.io/django-redis/latest/>`_ in order to be used with OMERO.web. We assume that Redis has already been installed. The `django-redis` package is now installed as part of the OMERO.web deployment. To configure the cache, run::
+  - `Redis <https://redis.io/>`_ requires `django-redis <https://niwinz.github.io/django-redis/latest/>`_ in order to be used with OMERO.web. We assume that Redis has already been installed. Select the step-by-step deployment guides from :doc:`install-web/web-deployment`. To configure the cache, run::
```
Comment (pwalczysko) [inline omero/sysadmins/unix/install-web.rst]: ``Select`` -> ``Follow``

**Comment by pwalczysko** [comment]
the change is fine.

**Comment by joshmoore** [comment]
Taking that as "good to merge". Thanks all.

**Comment by pwalczysko** [comment]
Maybe @will-moore can check the workflow as his laptop just died ?

**Comment by will-moore** [comment]
Sorry, I don't have a CentOS setup ready to use yet...

**Comment by pwalczysko** [comment]
@will-moore I was thinking about your Mac - install on Mac which you will be doing anyway - or do I get it wrong ?

**Comment by will-moore** [comment]
Testing https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/sysadmins/unix/install-web.html ?
OK, so I need to follow other install docs first to get omero-web installed.

**Comment by joshmoore** [comment]
Can I suggest https://docs.openmicroscopy.org/omero/5.6.0-rc1/ ?

**Pull request merged**